workflow/usage.rst

```diff
@@ -43,7 +43,7 @@ like this:
                     audit_trail:
                         enabled: true
                     marking_store:
-                        type: 'multiple_state' # or 'single_state'
+                        type: 'multiple_state' # or 'single_state', 'method'
```

Comment:
method was added in 4.3. we should be clear about that.

Reply:
What is the best way to write this?
In a comment next to this value ?
Above all this code ?